resources/lang/en/authorization.php

```diff
@@ -192,6 +192,15 @@
     ],
 
     'team' => [
+        'application' => [
+            'store' => [
+                'already_member' => "You're already part of the team.",
+                'already_other_member' => "You're already part of a different team.",
+                'currently_applying' => 'You have pending team join request.',
+                'team_closed' => 'The team is currently not accepting any join requests.',
+                'team_full' => "The team is full and can't accept any more members.",
```

Comment:
I forgot to mention, a (long) while ago it was discussed about always using single quotes for all the localization files and escaping as necessary? I forget the specific reason for going with single quotes only @peppy